tpp-rt/XsmmRunnerUtils.cpp

```diff
@@ -259,47 +262,6 @@ _mlir_ciface_xsmm_unary_scalar_invoke(const libxsmm_datatype dType,
   kernel(&param);
 }
 
-LIBXSMM_INLINE void matrix_copy_NC_to_NCNC(float *src, float *dst, int T, int N,
```

Comment:
This looks like an unrelated change...

Reply:
Yes, house cleaning. I can do it separately if needed.

Reply:
Easy NFC commit outside of PR.